aspnetcore/index.yml

```diff
@@ -0,0 +1,455 @@
+  ms.product: aspnet
+  ms.topic: hub-page
+  ms.collection: collection
+  author: WadePickett
+  ms.author: wpickett
+  ms.date: 02/26/2020
+
+# highlightedContent section (optional)
+# Maximum of 8 items
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  items:
+    # Card
+    - title: "Create an ASP.NET Core app on any platform in 5 minutes"
+      itemType: get-started
+      url: getting-started/index.md
+    # Card
+    - title: "ASP.NET Core Overview"
+      itemType: overview
+      url: introduction-to-aspnet-core.md
+    # Card
+    - title: "Video Tutorials"
+      itemType: video
+      url: https://www.youtube.com/playlist?list=PLdo4fOcmZ0oW8nviYduHq7bmKode-p8Wy
+    # Card
+    - title: "What's new in ASP.NET Core docs"
+      itemType: whats-new
+      url: whats-new/index.yml
+
+# conceptualContent section (optional)
+conceptualContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  title: Get Started # < 60 chars (optional)
+  summary: "Create your first ASP.NET Core app"
+  items:
+    # Card
+    - title: Create a web UI
+      summary: Create your first web UI in ASP.NET Core
+      links:
+        - url: getting-started/index.md
+          itemType: get-started
+          text: "Create an ASP.NET Core app on any platform in 5 minutes"
+        - url: tutorials/build-your-first-blazor-app.md
+          itemType: get-started
+          text: "Create a client-side web UI in C# with ASP.NET Core Blazor"
+    # Card
+    - title: Create a web API
+      summary: "Create your first web API in ASP.NET Core"
+      links:
+        - url: tutorials/first-web-api.md
+          itemType: get-started
+          text: "Create a RESTful web API with ASP.NET Core"
+        - url: tutorials/grpc/grpc-start.md
+          itemType: get-started
+          text: "Create a gRPC service with ASP.NET Core"
+    # Card
+    - title: Create a real-time web app
+      summary: "Create your first real-time web app in ASP.NET Core"
+      links:
+        - url: tutorials/signalr.md
+          itemType: get-started
+          text: "Create a real-time web app with ASP.NET Core SignalR"
+    # Card
+    - title: Create a data-driven web app
+      summary: "Create your first data-driven web apps"
+      links:
+        - url: tutorials/first-mvc-app.md
+          itemType: get-started
+          text: "Create a web app with ASP.NET Core MVC & SQL Server LocalDB"
+        - url: data/ef-rp/intro.md
+          itemType: get-started
+          text: "Razor Pages & Entity Framework Core"
+        - url: data/ef-mvc.md
+          itemType: get-started
+          text: "ASP.NET Core MVC with Entity Framework Core"
+        - url: https://docs.microsoft.com/en-us/learn/modules/build-web-api-net-core/
```

Comment:
@scottaddie , thanks, how do we handle links across repos's like that?   If I Just provided  the following does the system handle that just fine? :    learn/modules/build-web-api-net-core/

Reply:
Not sure if it will work or not, but try using "/learn/modules/build-web-api-net-core/" in this example.

Reply:
I can't find a way to get it to work across repos without using https://docs.microsoft.com.  Looking over other hub pages including the .NET one that is now live, they use the https://docs.microsoft.com for links across repos.   I will have to go with the prefix for now until someone can provide guidance for cross repo links that would not include that prefix.  Thanks for bringing it up though Scott.  It would seem like we could avoid the prefix.

Reply:
Use `https://docs.microsoft.com`, that's what we've been doing. Just drop the `/en-en/` culture.